boa/interpret.py

```diff
@@ -34,6 +35,7 @@
 from boa.rpc import json
 from boa.util.abi import Address
 from boa.util.disk_cache import DiskCache
+from boa.util.version import detect_version
```

Comment:
should we actually use the one from vvm?

Reply:
Charles is talking about this: https://github.com/vyperlang/vvm/pull/23